### ISSUE TITLE
Add help text via tooltips to various controls

### ISSUE THREAD

**Issue by fedarko** [biocore/qurro#225]
Using [Bootstrap's tooltips](https://getbootstrap.com/docs/4.3/components/tooltips/) seems like the easiest option (downside: will require adding Bootstrap JS, and popper.js, and jquery...)

This would be useful for explaining not-immediately-obvious behavior, e.g. when the "Categorical color scheme" is used, or explaining searching option behavior (#123), or detailing that tie-breaking behavior is undefined in autoselection (#189).

Could also use Bootstrap popovers for this, I guess.

edit -- small mockup from modifying bootstrap's docs. this looks super cool, and now that we're using jquery in #235 this should be totally doable in a future version of qurro (...not before monday but afterwards :P)

<img width="957" alt="Screen Shot 2019-12-12 at 10 43 37 PM" src="https://user-images.githubusercontent.com/4177727/70776072-e1abf480-1d30-11ea-8852-60af8d1abcea.png">

**Comment by fedarko** [comment]
Since we already have Popper.js / Bootstrap's JS set up in Qurro, this is really just a matter of

1) finding places in the app where we'd want to add tooltips
2) adding tooltip text
3) initializing tooltips when the page loads, via something like [this approach](https://getbootstrap.com/docs/4.3/components/tooltips/#example-enable-tooltips-everywhere).

Most of the work will likely be in steps 1 and 2.

(As a sidenote, it would be nice if we could add in a "?" glyph that is what you can hover over to see the tooltip -- would make sense from a usability perspective, rather than seemingly random placements of tooltips. This is kinda analogous to how Bandage works.)

<img width="1419" alt="Screen Shot 2020-02-25 at 6 13 01 PM" src="https://user-images.githubusercontent.com/4177727/75305227-5f44b400-57fa-11ea-8f45-5a2098208a27.png">

**Comment by fedarko** [comment]
Ok, these are set up pretty cleanly. Whenever we want to add a new "further info/help" icon, we can just add a HTML `<span>` element like so:

```html
<span class="questionmark" data-toggle="tooltip" data-placement="top"
    title="You can use this selection to change the sample metadata field used for the above plot's x-axis.">
</span>
```

... and the `qurro.css` will see the `questionmark` class and add in the icon, formatting, etc automatically. Furthermore, the JavaScript (in `setUpDOM()` in `display.js`) will use the `questionmark` class as a way to initialize the tooltips (this automatically takes care of step 3 in the above comment).

Here's what the above tooltip looks like in action:

<img width="1434" alt="Screen Shot 2020-02-25 at 10 23 44 PM" src="https://user-images.githubusercontent.com/4177727/75317656-63ce9400-581d-11ea-9193-3c7154b8ac64.png">

There's some formatting wonkiness regarding where you put the `<span>` element -- currently things are set up in such a way that we expect these elements to be child elements of `.input-group-text` `<label>` elements, and keeping these elements in this way makes them smaller than they would be outside of these elements. So uh TLDR if you wanna move the question marks to outside of the labels, you'd need to adjust size accordingly or they'd look weirdly large I guess. (in practice that'd be doable by just creating multiple `questionmark` classes with slightly different sizing -- not a super elegant solution but it would work.)

ANYWAY LONG STORY SHORT all that's left is just adding a bunch of these tooltips, and that shouldn't be too bad. It might be cool to have these be programmatically set based on the sort of data the user provides (e.g. the tooltip for the `Differential`/`Feature Loading` field would change based on that?), but that might be a stupid amount of work so I might just table that idea for now.

**Comment by fedarko** [comment]
Update: the one line of JS I added (`$(".questionmark").tooltip();`) broke the tests, life is suffering
